experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -68,14 +72,17 @@ export class TinyliciousClientInstance {
             runtimeFactory,
             false,
         );
-        return this.getRootDataObject(container);
+
+        const rootDataObject = await this.getRootDataObject(container);
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Why is this giving you an unsafe-return? It looks like you are returning a `FluidContainer` specifically, not an `any` type